src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Ball(){
+
+    }
```

Comment:
for why you need this empty constructor?

Reply:
I dont know) I deleted it